imod/mf6/simulation.py

```diff
@@ -638,7 +638,7 @@ def _merge_and_assign_exchange_budgets(self, cbc: GridDataset) -> GridDataset:
         exchange_budgets = cbc[exchange_names].to_array().sum(dim="variable")
         cbc = cbc.drop_vars(exchange_names)
         # "gwf-gwf" or "gwt-gwt"
-        exchange_key = exchange_names[0].split("_")[0]
+        exchange_key = exchange_names[0].split("_")[1]
```

Comment:
Why was this necessary? It changes behavior for gwf-gwt exchanges, was that required?

Reply:
Yes, since we have an extra '-' in de variable names.